packages/analytics/analytics-utilities/src/types/explore/requests.ts

```diff
@@ -1,13 +1,12 @@
 import { makeFilterable } from './util'
-import type { AbsoluteTimeRangeV4, MetricFilterTypesV2 } from './common'
+import type { AbsoluteTimeRangeV4, ExploreFilterTypesV2, RequestFilterTypeEqualsV2, RequestFilterTypeMetricV2, RequestFilterTypeWildcardV2 } from './common'
 import { queryableExploreDimensions, type ExploreFilter } from './advanced'
 
 export const queryableRequestDimensions = [
-  ...queryableExploreDimensions,
+  ...queryableExploreDimensions.filter(d => (d !== 'status_code' && d !== 'status_code_grouped')),
   'auth_type',
   'client_ip',
   'consumer_group',
-  'country_code',
```

Comment:
previous `iso_code` functionality is/will still be supported. we have decided to not support `country_coude` for this work